topic-operator/src/main/java/io/strimzi/operator/topic/K8sImpl.java

```diff
@@ -48,7 +48,7 @@ public K8sImpl(Vertx vertx, KubernetesClient client, Labels labels, String names
         vertx.executeBlocking(future -> {
             try {
                 KafkaTopic kafkaTopic = operation().inNamespace(namespace).create(topicResource);
-                LOGGER.debug("KafkaTopic {} created with version {}->{}",
+                LOGGER.debugOp("KafkaTopic {} created with version {}->{}",
```

Comment:
Why does this not belong to any reconciliation? Sounds like deleting or creating topics should always belong to a particular reconciliation. Applies to most logging done here.